recipes/ius.rb

```diff
@@ -0,0 +1,23 @@
+#
+# Cookbook:: osl-git
+# Recipe:: ius
+#
+# Copyright:: 2020, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_recipe 'yum-ius'
+
+git_client 'ius git224' do
+  package_name 'git224'
```

Comment:
I think it might be a good idea to make this an attribute which defaults to this in case we want to install something different.